packages/govern-server/src/api-graphql/resolvers.ts

```diff
@@ -20,12 +20,22 @@ export default function resolvers(govern: GovernCore): IResolvers {
     Dao: {
       async registryEntries(parent) {
         return Promise.all(
-          parent.registryEntries.map(async ({ id }: { id: string }) =>
-            govern.registryEntry(id)
-          )
+          parent.registryEntries
+            .map(async ({ id }: { id: string }) => {
+              const entry = await govern.registryEntry(id)
```

Comment:
This should be moved to the ``GovernCore`` and I still don't get the ``Dao.registryEntries`` cause it would be ``Registry.entries``. (already discussed on discord quickly with @0xGabi)

Reply:
> This should be moved to the `GovernCore`

Moving it to `GovernCore` would mean querying all the registry entries and their data. The idea here is to find the right balance between querying everything in a giant query, or doing a big amount of small queries to resolve individual fields.

> I still don't get the `Dao.registryEntries` cause it would be `Registry.entries`. (already discussed on discord quickly with @0xGabi)

`RegistryEntry` contains a DAO (executor) and a queue, so we could have any amount of `RegistryEntry`. Let me know if that answers your question, I’m not sure it does :smile:

Reply:
I think ``parent.registryEntries`` can easily get passed to the ``GovernCore`` nothing to change at the request logic here.. just moving the stuff :-) 

> RegistryEntry contains a DAO (executor) and a queue, so we could have any amount of RegistryEntry. Let me know if that answers your question, I’m not sure it does

Yes, we could have any amount of registry entries in the whole system but we only have one entry per DAO. This as I see is a child of the Dao GraphQL Type which would be wrong :-)